File: OnePointFour.java
import java.util.Arrays;

public class OnePointFour {

    // check two strings are anagrams or not

    // solution 1
    private static boolean checkAnagram(String a, String b) {
        char [] aArray = a.toCharArray();
        char [] bArray = b.toCharArray();

        Arrays.sort(aArray);
        Arrays.sort(bArray);
        return new String(aArray).equals(new String(bArray));
    }

    public static void main(String ar[]) {
        System.out.println(checkAnagram("abc", "bca"));
        System.out.println(checkAnagram("abc", "bcaa"));
    }
    
}

File: CrackingTheCodingInterviews/Arrays/OnePointFour.py

class OnePointFour:
    def __init__(self) -> None:
        pass

    # check two strings if they are anagrams 
    def check(self, string1, string2):
        
        print(sorted(string1))
        print(sorted(string2))
        return sorted(string1) == sorted(string2)


one = OnePointFour()
print(one.check("abc","cba"))
print(one.check("abc","cbaa"))
print(one.check("abc","pqr"))
